modules/vsphere-problem-detector-operator.adoc

```diff
@@ -0,0 +1,32 @@
+// Module included in the following assemblies:
+//
+// *  operators/operator-reference.adoc
+
+:operator-name: vSphere Problem Detector Operator
+
+[id="vsphere-problem-detector-operator_{context}"]
+= vSphere Problem Detector Operator
+
+[discrete]
+== Purpose
+
+The {operator-name} checks the cluster for the most common installation and misconfiguration issues.
+
+[discrete]
+== Project
+
+link:https://github.com/openshift/vsphere-problem-detector[vsphere-problem-detector]
```

Comment:
Please request approval from our Support team for GitHub links.

Reply:
We should remove this GH link. As Paul states, it would require approval by Support but those requests are minimal and should be avoided.

Reply:
Hi Gents, I'm not spotting how I approach the support folks.  I looked in the docs user manual, but I can't find it.  Every operator provided by Red Hat includes a link to the repo, so I expect the request to be approved--regardless, I still can't figure out how to make that request.

Reply:
@sferich888, is this an acceptable GitHub link? 

@mikemckiernan, GitHub links are [forbidden](https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#links-to-external-websites). Eric Rich is willing to grant exceptions in extraordinary circumstances.